docs/en-us/manual/connection.md

```diff
@@ -149,24 +149,34 @@ For specific differences, you can read the [source code](https://github.com/MaaA
 
 ### MuMu Screenshot Enhanced Mode
 
-Only support official China or Ark Edition MuMu 12 V4.0.0 and later versions, and close MuMu's background keep alive. Global Edition are not supported now. <!-- V3.8.13 -->
+Only support official China or Ark Edition MuMu 12 V4.0.0 and later versions, and close MuMu's background keep alive. Global Edition are not supported now. <!-- official V3.8.13 -->
```

Comment:
I don't think global understands the term official, as the Chinese version.
It may seem a bit racist, but I think for EN, Chinese would be better compared to official.